charts/atlantis/values.yaml

```diff
@@ -8,6 +8,7 @@
 
 # Replace this with your own repo allowlist:
 orgAllowlist: <replace-me>
+#githubAppOrg: myorgname
```

Comment:
Could we move this to githubApp?

Reply:
Unfortunately we can't be under githubApp, because the org needs to be set up before the github app is created and set up in atlantis

Reply:
Hi @tampakrap, can you please clarify why it wouldn't work? We can still set the value on the `ATLANTIS_GH_ORG`, it's just a different name for the variable.